File: minwook/7주차/없는_숫자_더하기.java
import java.util.*;
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int[] numArr = {0,1,2,3,4,5,6,7,8,9};
        for(int n : numbers){
                if(Arrays.stream(numArr).anyMatch(i -> i == n)){ // 숫자 존재할 경우 다 0으로 치환
                    numArr[n] = 0;
                }
        }
        for(int n : numArr){
            answer += n;
        }
        return answer;
    }
}

File: yongkeun/7week/없는숫자더하기.js
function solution(numbers) {
  const zeroToNine = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
  let answer = 0;
  for (let i = 0; i < zeroToNine.length; i++) {
    if (!numbers.includes(zeroToNine[i])) {
      //numbers 배열에 0이 없는지, 1이 없는지,
      answer += zeroToNine[i]; //없으면 더하기
    }
  }
  return answer;
}
